src/rust/receiver-mock/src/router/mod.rs

```diff
@@ -577,6 +577,12 @@ pub async fn handler_logs_count(
     HttpResponse::Ok().json(LogsCountResponse { count })
 }
 
+pub async fn handler_dump(body: web::Bytes) -> impl Responder {
```

Comment:
I wonder if there is no other type that can be used as an argument here to get the stringified body already at the argument level.

Reply:
I couldn't see any obvious way of doing this in the actix docs.

Reply:
Me neither: https://docs.rs/actix-web/latest/actix_web/web/index.html